ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -110,6 +110,10 @@ class DemoListViewController: DemoTableViewController {
         UserDefaults.standard.set(demo.title, forKey: DemoListViewController.lastDemoControllerKey)
     }
 
+    override func tableView(_ tableView: UITableView, canFocusRowAt indexPath: IndexPath) -> Bool {
```

Comment:
I’m guessing this is for focus support? And we always want to be true right?

Reply:
yes